server/src/routes/api/v1/podcastVotesRouter.js

```diff
@@ -0,0 +1,105 @@
+import express from 'express'
+import objection from 'objection'
+const { ValidationError } = objection
+
+import { Vote } from '../../../models/index.js'
+
+const podcastVotesRouter = new express.Router({ mergeParams: true })
+
+podcastVotesRouter.get('/', async (req, res) => {
+  const { podcastId } = req.params
+  let userId
+  if (req.user) userId = req.user.id
+
+  try {
+    let votes = {}
+    votes.userVotes = await Vote.query()
+      .where('podcastId', podcastId)
+      .andWhere('userId', userId)
+      .first()
+    votes.totalVotes = await Vote.query()
+      .where('podcastId', podcastId)
+      .sum('value as value')
+      .first()
```

Comment:
nice job with sum. do we really need the `.first()` here? if so that's fine, but it does look odd. Ensure that this is returning one number that represents the total votes for a podcasts.

Reply:
The `.first()` returns only an object instead of an "array" of one object. Not "essential" but made sense for what we're trying to accomplish.